tests/unit/test_typeset_default.py

```diff
@@ -177,7 +180,7 @@ def test_contains(name, series, contains_type, member):
 
 inference_map = {
     "int_series": Numeric,
-    "categorical_int_series": Numeric,
+    "categorical_int_series": Categorical,

```

Comment:
this and the float one are a bit odd, I didn't see any changes that should affect this test. It's not an issue but odd nonetheless.